.eslintrc

```diff
@@ -0,0 +1,213 @@
+ecmaFeatures:
```

Comment:
@rzelayafavila : Once you extend google, do you need any of these, or are you now disabling things that are actually in accordance with google styles? For example, we're getting `no-empty-label` errors because that is being set, but doesn't exist anymore. Instead the new one is `no-labels`